README.md

```diff
@@ -1 +1,159 @@
-# to-do-list
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+ 
+
+  <h3><b>Awesome Books</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 To Do List <a name="about-project"></a>
+
+**To Do List** is a tool for managaging your everyday tasks. You add your tasks and upon completion, you check to indicate its done.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li> JS </li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Add button]**
+- **[Remove button]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- You can see the live demo of this project [here](https://dazehinn.github.io/to-do-list/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- git (for version control)
+- web browser
+Have text editor installed. (VSCode, sublime, atom)
+- (optional) Have CSS,HTML & JS Linters installed and configured. For further information, visit [this link](https://github.com/microverseinc/linters-config/blob/master/README.md) to get better understanding on each linters used on this project.
+
+### Setup
+
+* Clone this repository to your desired folder
+* open the index.html file using a web browser
+
```

Comment:
- [ ] when following the setup instructions the page shown is just a blank screen with the text "This is a ToDo app"